src/i18n/locales/cs/translation.json

```
@@ -741,7 +741,7 @@
     "success": "Úspěch",
     "error": "Chyba",
     "settings": "Nastavení",
-    "websites": "Webové stránky",
+    "connected_sites": "Connected Sites",
```

Rename the key, but leave the translation empty. 

```suggestion
    "connected_sites": "",
```

---

Apply this for all translations, please.

---

ok sure